.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['7.4', '8.0', '8.1', '8.2']
-        nextcloud-versions: ['master']
+        nextcloud-versions: ['master', 'stable25']
```

Comment:
let's include just one job or we will go from 4 test jobs to 8. 5 should be plenty.